### DISCUSSION TITLE
How to reset the reported energy for Xiaomi power plug ZNCZ04LM

### DISCUSSION THREAD

**Discussion by michaelmell** [Koenkk/zigbee2mqtt#17175] [Devices]
Dear all,

I have the following smart power plug from Xiaomi:
https://www.zigbee2mqtt.io/devices/ZNCZ04LM.html#xiaomi-zncz04lm

It currently reports a total "energy" of 9.78kWh:
![image](https://user-images.githubusercontent.com/45131936/228515573-99c54fd0-9a3d-4560-a09e-739f284576cb.png)

I would like to reset its value to 0, because I have moved the plug to a new device, so I want to restart counting from zero. Is this possible?

Thanks in advance,
Michael

**Comment by Oli-BY** [comment]
did you found a solution?

**Comment by NewFolk** [comment]
Also looking for a solution...
Thanks in advance for sharing!

**Comment by keithr59** [comment]
I found this article but I haven't tried it myself. 

https://smarthomescene.com/guides/how-to-reset-smart-plugs-energy-total-in-zigbee2mqtt-and-zha/

**Reply by NewFolk** [reply]
Thank you. Unfortunately this manual is for another devices :(
